app/controllers/openid_connect/logout_controller.rb

```diff
@@ -17,6 +24,19 @@ def index
         success: result.success?,
```

Comment:
Do we need to update the handling of the analytics / attempts API? In the new workflow, should those only be logged at the point that the user confirms logout?

Reply:
I had a go at a more significant re-do of analytics by creating a few more events related to OIDC Logout and separating them from "Logout Initiated"